tools/maven/suppressions.xml

```diff
@@ -82,4 +82,7 @@ under the License.
 		<suppress
 			files="org[\\/]apache[\\/]flink[\\/]formats[\\/]avro[\\/]glue[\\/]schema[\\/]registry[\\/]User.java"
 			checks=".*"/>
+		<!-- suppress check for copied hive code -->
+		<suppress files="HiveConf.java" checks="[a-zA-Z0-9]*"/>
```

Comment:
Make this less specific because otherwise we might forget to drop it in the future. We can simply do a `org.apache.hive` exclusion.